features/plain_features/report_stack_frames.feature

```diff
@@ -14,8 +14,8 @@ Scenario Outline: Stack frames can be removed
   And the request used the "Ruby Bugsnag Notifier" notifier
   And the request used payload v4 headers
   And the request contained the api key "a35a2a72bd230ac0aa0f52715bbdc6aa"
-  And the "file" of stack frame 0 equals "/usr/src/app/stack_frame_modification/initiators/<initiator>.rb"
```

Comment:
You should instead update the features to raise the exception so that bugsnag doesnt appear in the stacktrace

Reply:
I've updated the scenarios to separate the tests that require this change and those that don't.